admin_console/web/js/assignPasswordRegex.js

```diff
@@ -0,0 +1,62 @@
+function passwordValidationRulesSaveButtonPressed()
+{
+	let regexObject = jQuery.parseJSON("{}");
+	jQuery("fieldset[class=regex-rule]").each(function() {
+		let regex = jQuery(this).find("input[id=regex]").val();
+		let description = jQuery(this).find("input[id=description]").val();
+		regexObject[regex] = description;
```

Comment:
it happens only if regex was entered?

Reply:
Occurs after clicking 'Save' in the 'Edit Rules' window